.github/workflows/eol-outdated-js-scan.yml

```
@@ -9,53 +10,13 @@ concurrency:
-
-      - name: Depcheck
-        uses: fylein/eol-outdated-js-scan-action@master
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          suppression: 'suppression.xml'
+    with:
+      npm-run-cmd: 'npm ci'
+      suppression: 'suppression.xml'
+    secrets:
+      github-token: ${{ secrets.GITHUB_TOKEN }}
+      nvd-api-key: ${{ secrets.NVD_API_KEY }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)